src/librustc/middle/dead.rs

```diff
@@ -464,21 +460,19 @@ impl<'a, 'tcx> DeadVisitor<'a, 'tcx> {
         // method of a private type is used, but the type itself is never
         // called directly.
         let impl_items = self.tcx.impl_items.borrow();
-        match self.tcx.inherent_impls.borrow().get(&self.tcx.map.local_def_id(id)) {
-            None => (),
-            Some(impl_list) => {
+        if let Some(impl_list) =
+            self.tcx.inherent_impls.borrow().get(&self.tcx.map.local_def_id(id)) {
                 for impl_did in impl_list.iter() {
```

Comment:
Same here -- the block starting at the above line should be unindented one level